Messages/MessageElement.php

```diff
@@ -67,7 +67,9 @@ public function __construct($title, $subtitle, $image_url = '', $buttons = [], $
         $this->url = $url;
         $this->image_url = $image_url;
         $this->buttons = $buttons;
-        $this->default_action = $default_action;
+        if (!empty($default_action)) {
+            $this->default_action = $default_action;
+        }
```

Comment:
I think lines 70,71,72 are not required

can be left as before `$this->default_action = $default_action;`